### PR TITLE
Inject player config function

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(3) As Rise Player, I inject a configuration function to the window's global object and the function is available before any other javascript executes](https://trello.com/c/1whqqGhp/4880-3-as-rise-player-i-inject-a-configuration-function-to-the-windows-global-object-and-the-function-is-available-before-any-other-j)

File: src/main/viewer/preload/player-configuration.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const platform = require("rise-common-electron").platform;
+const commonConfig = require("common-display-module");
+const gcs = require("../../player/gcs");
+
+const displaySettings = commonConfig.getDisplaySettingsSync();
+
+window.getRisePlayerConfiguration = function() {
+  const playerType = commonConfig.isBetaLauncher() ? "beta" : "stable";
+  const playerVersion = commonConfig.getLatestVersionInManifest();
```
Comment (tejohnso) [inline src/main/viewer/preload/player-configuration.js]: I don't think viewer should be making backend display module calls directly. Why aren't they all being provided at module instantiation time like displaySettings?

File: src/main/viewer/preload/player-configuration.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  const os = platform.getOSDescription();
+
+  const displayId = displaySettings.displayid;
+  const data = gcs.getCachedFileContentsSync(`risevision-display-notifications/${displayId}/content.json`);
+
+  let companyId = "";
+  if (data && data.content && data.content.schedule) {
+    companyId = data.content.schedule.companyId;
+  }
+
```
Comment (tejohnso) [inline src/main/viewer/preload/player-configuration.js]: I think we should send a message back out to player to indicate that the configuration function was called.

Could come in handy.

File: src/main/viewer/preload/player-configuration.js (hunk 1 above)
Comment (andrecardoso) [reply src/main/viewer/preload/player-configuration.js]: My concern was removing the display id from the schedule and clearing the company id, but I tested and it doesn't happen as viewer is loaded in this case.

File: src/main/viewer/preload/player-configuration.js (hunk 2 above)
Comment (andrecardoso) [reply src/main/viewer/preload/player-configuration.js]: 👍

**Pull request merged**